src/main/java/org/projecteka/hiu/Application.java

```diff
@@ -10,13 +10,16 @@
 
 import java.sql.Connection;
 import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
 
 public class Application {
     public static void main(String[] args) {
-        System.out.println("Creating schema for health information user");
+        System.out.println("Connecting...");
         try {
             Connection connection = openConnection();
             Database database = DatabaseFactory.getInstance().findCorrectDatabaseImplementation(new JdbcConnection(connection));
+            System.out.println("Creating schema for health information user");
             Liquibase liquibase = new liquibase.Liquibase("liquibase.xml", new ClassLoaderResourceAccessor(), database);
             liquibase.update(new Contexts(), new LabelExpression());
         } catch (Exception e) {
```

Comment:
There seems to be a lot of code in the "Exception" block. Ideally all this should be in a separate class/method.. 
getJDBCConnectionURL(), and checkAndCreateDB(), etc. This will also allow for writing tests for this separate class/method (rather than being inside Main() ).